resources/views/Containers/ItemLists/ItemList1.twig

```diff
@@ -1,27 +1,31 @@
 {{ component( "Ceres::Containers.Components.ContainerItemList" ) }}
+{{ component( "Ceres::Containers.Components.LastSeenItemList" ) }}
 
 {% set listType = config("Ceres.item.lists.1.list_type") %}
 {% set tagIds = config("Ceres.item.lists.1.tag_ids") | split(",") %}
 {% set itemListId = 1 %}
 {% set isHome = services.template.isHome() %}
 
-{% if listType == "last_seen" %}
-    {% set itemList = services.itemLoader.loadForTemplate('Ceres::ItemList.ItemListView', itemLoaderPresets.lastSeenItemsList, {'items': itemsPerPage}).documents %}
-{% elseif listType == "cross_selling" %}
+{% if listType == "cross_selling" %}
     {% set itemList = item.CrossSellingItems.documents %}
 {% elseif listType == "tag_list" %}
 	{% set itemList = services.itemLoader.loadForTemplate('Ceres::ItemList.ItemListView', itemLoaderPresets.tagList, {'tagIds': tagIds}).documents %}
 {% endif %}
 
-{% if itemList is not empty %}
-    {% if not isHome %}
-        <hr>
-    {% endif %}
+{% if not isHome %}
```

Comment:
@lks-mtzn maybe add a check here for itemList || listType == "last_seen", to avoid `<hr>` elements without list?